gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageOptions.java

```diff
@@ -37,7 +37,7 @@
   public static final boolean ENABLE_GRPC_DEFAULT = false;
 
   /** Default setting to prefer DirectPath for gRPC. */
-  public static final boolean DIRECT_PATH_PREFFERED_DEFAULT = true;
+  public static final boolean DIRECT_PATH_PREFERRED_DEFAULT = true;
```

Comment:
Do we want to make the false by default? So whoever uses our client doesn't have to override themselves?

Reply:
Switching to Cloudpath should be Opt-in IMO. As most customers would benefit from Directpath.

Swithcing to cloudpath would be an option only in case of RVIP configurations. Customers who don't use RVIP with VPC-SC would be penalised with a default of cloudpath

Reply:
If we communicate to customer who will use with RVIP, then I guess it is okay. Not sure how communication to customer in preview is going to work. Otherwise the customer who use RVIP may spend lot of time trying to debug something which anyway wouldn't work.

Also I am assuming in VPC-SC tests, we should be able to override this to false in their config.